src/conversions.cpp

```diff
@@ -237,4 +237,28 @@ void getRobotTrajectory(const ::drake::trajectories::Trajectory<double>& drake_t
     t_prev = t;
   }
 }
+
+std::string replaceSTLWithOBJ(const std::string& input)
+{
+  std::string result = input;
+  const std::string lower_case_stl = ".stl";
+  const std::string upper_case_stl = ".STL";
+  const std::string replacement = ".obj";
```

Comment:
Why didn't we think of to use `constexpr` here? Just take it as a question bc i haven't use `constexpr` before.